contracts/clients/src/helpers/hashBundle.ts

```diff
@@ -0,0 +1,50 @@
+import { BigNumberish, ethers } from "ethers";
+import { Bundle, Operation } from "../signer";
+
+type BundleWithoutSignature = {
+  senderPublicKeys: [BigNumberish, BigNumberish, BigNumberish, BigNumberish];
+  operations: Omit<Operation, "gas">;
+};
+
+export function hashBundle(bundle: Bundle, chainId: number): string {
+  if (bundle.operations.length !== bundle.senderPublicKeys.length) {
```

Comment:
Thought the room for incorrect arguments was larger in a client module function. Added this basic check but held off on adding anymore, like validating more of the bundle, or chain id etc. Let me know if anyone thinks otherwise